src/postgres/src/backend/utils/misc/pg_yb_utils.c

```diff
@@ -3964,3 +3964,81 @@ YbReadWholeFile(const char *filename, int* length, int elevel)
 	buf[*length] = '\0';
 	return buf;
 }
+
+/*
+ * For backward compatibility, this function dynamically adapts to the number
+ * of output columns defined in pg_proc.
+ */
+Datum
+ycql_stat_statements(PG_FUNCTION_ARGS)
+{
+  FuncCallContext *funcctx;
+  int expected_ncols = 9;
+  static int ncols = 0;
+
+  if (ncols < expected_ncols)
+    ncols = YbGetNumberOfFunctionOutputColumns(8067 /* yb_servers function
```

Comment:
`ycql_stat_statements` instead of `yb_servers`. Also, probably better to keep this function in yb_auh.c